resources/zib/zib-PharmaceuticalProduct.xml

```diff
@@ -0,0 +1,176 @@
+      <alias value="FarmaceutischeVorm" />
+      <binding>
+        <strength value="example" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.9.7.8--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-pharmaceuticalproduct-v2.1.2-2020EN" />
+        <map value="NL-CM:9.7.19931" />
+        <comment value="PharmaceuticalForm" />
+      </mapping>
+    </element>
+    <element id="Medication.ingredient">
+      <path value="Medication.ingredient" />
+      <short value="Ingredient" />
+      <definition value="A product contains one or more active substances and excipients. These are usually determined by the product code. For medication prepared or compounded by the local pharmacy, each ingredient must be entered separately.&#xD;&#xA;&#xD;&#xA;The active substances play an important role, as they: a) determine the pharmacotherapeutic effect of the medication and b) serve as the basis for the indication of the strength of the medication (e.g. 200mg)." />
+      <alias value="Ingredient" />
+      <mapping>
+        <identity value="zib-pharmaceuticalproduct-v2.1.2-2020EN" />
+        <map value="NL-CM:9.7.19932" />
+        <comment value="Ingredient" />
+      </mapping>
+    </element>
+    <element id="Medication.ingredient.item[x]">
+      <path value="Medication.ingredient.item[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Medication.ingredient.item[x]:itemCodeableConcept">
+      <path value="Medication.ingredient.itemCodeableConcept" />
+      <sliceName value="itemCodeableConcept" />
+      <short value="SubstanceCode" />
+      <definition value="Active substance or excipient. Here, the same codes can be used as for the ProductCode (for dilutions and compounds in particular), but now, the ATC, SSK and SNK codes can also be used to indicate a substance (to list ingredients of local products prepared by the pharmacy). &#xD;&#xA;* GTIN International Article Number &#xA;* KNMP article number &#xA;* Trade product code (HPK) &#xA;* Prescription code (PRK) &#xA;* Generic product code (GPK) &#xA;* ATC (anatomic therapeutic classification) &#xA;* SSK (substance name code with route of administration) &#xA;* SNK (substance name code) &#xD;&#xA;For further explanation about ATC see https://www.whocc.no/atc/structure_and_principles/" />
+      <alias value="IngredientCode" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.13--20200901000000" />

```

Comment:
Known issue about the IngredientCodeSSKCodelist 'Extensible' binding

https://bits.nictiz.nl/browse/ZIB-1453

Reply:
Idem